src/browser/modules/D3Visualization/lib/visualization/components/layout.ts

```diff
@@ -35,11 +35,7 @@ import Relationship from './Relationship'
 import VizNode from './VizNode'
 
 type ForceLayout = {
-  update: (
-    graph: Graph,
-    size: [number, number],
-    animate?: boolean
-  ) => Simulation<VizNode, Relationship>
+  update: (graph: Graph, animate?: boolean) => Simulation<VizNode, Relationship>
```

Comment:
`animate?` is not used anywhere. Can you also remove it?

Reply:
Nice catch! It is actually used, since when I tried to remove it I see that it complains in Vizualization.ts row 324 where it is used. I guess it is just mapped to a different name (precompute), though it feels strange that typescript doesn't complain so probably need to specify typescript better here?? However since this is out of scope I'll leave as it is for this PR.